cmd/krew/cmd/root.go

```diff
@@ -113,9 +144,18 @@ func preRun(cmd *cobra.Command, _ []string) error {
 			klog.Warningf("You may need to clean them up manually. Error: %v", err)
 		}
 	}
+
 	return nil
 }
 
+func showUpgradeNotification(*cobra.Command, []string) {
+	if latestTag == "" || latestTag == version.GitTag() {
```

Comment:
I think I've mentioned this before: Let's parse both of these into semver, and actually do `LessThan` check.

Reply:
Yes, I also explained why I think that mere string comparison is better:

We only allow the version numbers of krew releases to advance, so that it is equivalent to check for string equality vs. semver less-than comparison.

But if you feel so strongly about it, I can change that.

Reply:
I feel like we're gonna see upgrade notifications etc in CI tests. 
If it's not a big deal, let's just do that. We already have the code, and we'd simply skip if it fails to parse as semver.